debian/control

```diff
@@ -7,5 +7,5 @@ Standards-Version: 3.9.6.1
 
 Package: gpgmailer
 Architecture: all
-Depends: adduser (>= 3.11), confighelper (>=0.8), python-gnupg, python-daemon
+Depends: adduser (>= 3.11),parkbench-common (>=0.8),postfix | mail-transfer-agent,python-gnupg,python-daemon
```

Comment:
Although this is probably an existing issue, python-daemon should be listed before python-gnupg.